contrib/gofiber/fiber.v2/fiber.go

```diff
@@ -48,16 +48,12 @@ func Middleware(opts ...Option) func(c *fiber.Ctx) error {
 		// pass the span through the request UserContext
 		c.SetUserContext(ctx)
 
-		resourceName := c.Path()
-		if resourceName == "" {
-			resourceName = "unknown"
-		}
-		resourceName = c.Method() + " " + resourceName
-		span.SetTag(ext.ResourceName, resourceName)
-
 		// pass the execution down the line
 		err := c.Next()
 
+		// Setting the route path as resource name
```

Comment:
Personally I think this comment is not needed. It's pretty obvious what's going on.